docker-bake.hcl

```diff
@@ -305,6 +305,7 @@ target "virtual-cloud-tools" {
                         "libxcrypt-compat", 
                         "azure-cli",
                         "awscli",
+                        "openssh-clients",
```

Comment:
:angry: You people intentionally try to trigger me with this unordered list, don't you? *checks-whether-the-alphabet-changed-while-away* :eyes:

Reply:
To my defense, it was not ordered in the first place and this addition was actually alphabetically ordered with a part of the list 😄